keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt

```diff
@@ -24,8 +24,8 @@ import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
 
 @ExtendWith(SpringExtension::class)
 @SpringBootTest(
-  classes = [KeelApplication::class, MockEurekaConfiguration::class],
-  webEnvironment = MOCK
+    classes = [KeelApplication::class, MockEurekaConfiguration::class],
+    webEnvironment = MOCK
```

Comment:
more weird indentation shenanigoats in this file, doesn't look like there are any actual changes